.tool-versions

```diff
@@ -0,0 +1,3 @@
+maven 3.8.2
```

Comment:
I think `direnv` is missing from here.  It refuses to work for me otherwise, at least.

Reply:
Added in 8987d6eb25d2913555050c2e2ea8b539561f3f0d